Purchases/Purchasing/OfferingsFactory.swift

```diff
@@ -27,9 +27,17 @@ class OfferingsFactory {
             var dict = dict
             if let offering = createOffering(withProducts: products, offeringData: offeringData) {
                 dict[offering.identifier] = offering
+                if offering.availablePackages.isEmpty {
+                    Logger.warn(Strings.offering.offering_empty(offeringIdentifier: offering.identifier))
+                }
```

Comment:
new behavior, show something in the console if there are no packages for an offering.

Reply:
I feel like this is a huge upgrade.